xkcd.py

```diff
@@ -246,21 +253,26 @@ async def _send_xkcd(self, room_id: RoomID, xkcd: XKCDInfo) -> None:
             content["license_url"] = "https://xkcd.com/license.html"
             await self.client.send_message(room_id, content)
         else:
-            await self.client.send_text(room_id, text=f"{xkcd.num}: **{xkcd.title}**",
-                                        html=f"{xkcd.num}: <strong>{xkcd.safe_title}</strong>")
```

Comment:
Not sure if this should take over the existing non-inline form or be a new format option 🤔

It's slightly different from either of the existing options

Reply:
Yeah I wondered this myself too. I can make a 3rd variant. Instead of having two independent flags that are mutually exclusive, it would make more sense to me to have a flag with 3 options. I'll rework this

Reply:
Yeah a new field probably makes most sense. You can automatically migrate configs in the `do_update` method with something like

```python
if "inline" in self:
  helper.base["post_mode"] = "inline" if self["inline"] else "split"
else:
  helper.copy("post_mode")
```

(assuming the new field is called `post_mode` with values `inline`, `split` and `caption`)

Reply:
Perfect, updated to use this. Thanks